lte/gateway/c/oai/include/mme_app_ue_context.h

```diff
@@ -369,8 +360,6 @@ typedef struct ue_mm_context_s {
 
   /* TODO: Add TAI list */
```

Comment:
This TODO is not needed any more.

Reply:
The commented code above can also be removed as part of this change.

Reply:
Removed in PR, 766